.github/workflows/test.yaml

```
@@ -800,6 +800,7 @@ jobs:
           - nodejs
           - cloudflare-workers
           - vercel-edge-functions
+          - vercel-cli
```

Let's rename this to:
```suggestion
          - vercel-serverless-functions-via-cli
```

---

(We might want to add `via git` later or other deployment mechanisms)